trustmanager/keystore.go

```diff
@@ -328,26 +327,22 @@ func GetPasswdDecryptBytes(passphraseRetriever notary.PassRetriever, pemBytes []
 			err    error
 		)
 		passwd, giveup, err = passphraseRetriever(name, alias, false, attempts)
+		if attempts > 10 {
```

Comment:
Should this check go before we call `passphraseRetriever`, since we've determined that we have tried enough times and don't actually use `passwd`?